quiz.js

```diff
@@ -10,66 +9,53 @@
 
 // The character to use should be from user input in a <input type="text"> field in the DOM.
 
+// It accepts a single object as an argument. The object should have two key/value pairs.
 
 var tree = {
 	height: userHeight,
```

Comment:
Is it necessary to create these properties or will they be created when you actually have data in your `checkInput` function?